unit-tests-jest/numbers.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers) {
+  return numbers.filter((n) => (n % 2 === 0)
+  );
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ * If not a number returns NaN.
+ * If a decimal or negative number returns undefined.
+ */
+export function toDollars(amount) {
+  if (isNaN(amount)) return NaN;
+  if (!Number.isInteger(amount) || !isFinite(amount) || amount < 0) return undefined;
+  return `$${amount}.00`;
```

Comment:
This won't work for fractional amounts. You need to add that case to your tests.